go

```diff
@@ -0,0 +1,9 @@
+{
+  default-service-strategy: allow,
+  services: {
+    iam: {
+      type: allow
+    }
+  }
+}
+ run . iam org-policy update -
```

Comment:
did you commit this file by accident?

Reply:
Fixed in 128baae8, looks it was the result of an incorrect pipe